### DISCUSSION TITLE
Assembly: NFApp2 (1.0.0.0) needs assembly 'mscorlib' (1.14.3.0) Error: a3000000

### DISCUSSION THREAD

**Discussion by RolfSste** [nanoframework/Home#1408] [Q&A] [answered]
I'm new to nanoFramework and I want to run the simple "Hello from nanoFramework!" example on an STM32F407 DISCOVERY board. I'm using .NET nanoFramework Extension Version: 2022.3.0.65 on VisualStudio Community 2022. 

How can I resolve this problem ?

**Comment by TerryFogg** [comment]
Use Manage nuget packages for solution and update the installed nugets to the latest

**Reply by RolfSste** [reply, marked as answer]
I thought installed
.NET nanoFramework VS Extension v2022.3.0.65 [Latest](https://github.com/nanoframework/nf-Visual-Studio-extension/releases/latest)
@[nfbot](https://github.com/nfbot) nfbot released this 24 Oct 17:26
v2022.3.0.65
851a8fd

is the latest version ?

**Comment by TerryFogg** [comment]
No, update to latest nuget package


<img width="501" alt="image" src="https://github.com/nanoframework/Home/assets/12385835/68fc1644-016a-4504-a439-613ad2d91105">

<img width="1087" alt="image" src="https://github.com/nanoframework/Home/assets/12385835/8f2b1fbe-9f99-4f62-9e91-91b185d668b1">

**Reply by RolfSste** [reply]
Thank you for your help, after some twiddling I got it to work.

**Reply by koldomon** [reply]
Had the same problem. Updated to latest nuget Package, restarted VS, "Build" multiple times, "Erase Device Deployment Areas" , but error remains. 

Got it to work after "Clean" and "Rebuild" Project.
